test/utils/secret.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+type Secret struct {
+	TestSecretsInterface bool
+}
+
+// TODO: rename to a more generic name such as TestContentInterfaces
+// to allow having both secret seeding and config interfaces inside
+func TestSecret(t *testing.T, providerPath, consumerName, consumerSecretPath string, conf Secret) {
```

Comment:
Why some of the configurations are in the config object, some others as function arguments?